content/docs/tools-ui/component-library/uidev-card-widget.md

```diff
@@ -30,7 +30,7 @@ Cards allow for a certain amount of customization. The basic layout of a card in
     (Optional) Cards can have their own unique icon related to their purpose. For the full list of icons, see [O3DE component icons](/docs/tools-ui/icon-assets/uidev-component-icons/).
    
     {{< note >}}
-Two icons are required for new components:
+Two icons are required for new widgets:
```

Comment:
I think this word should still be `components` because the new icons being referenced are meant to describe the component itself. Only one of the icons is actually visible in the card widget itself, the other is for the viewport.